testsuite/gateways/apicast/operator.py

```diff
@@ -116,10 +116,7 @@ def environ(self) -> Properties:
         return self._environ
 
     def reload(self):
-        self.openshift.do_action("delete", ["pod", "--force",
-                                            "--grace-period=0", "-l", f"deployment={self.deployment}"])
-        # pylint: disable=protected-access
-        self.openshift.wait_for_ready(self.deployment)
+        self.openshift.deployments.rollout(self.deployment)
```

Comment:
these are not equal, while I am in favor of new suggestion this may be risky

Reply:
In which sense? I believe they are equal. The KubernetesDeployments rollout method should be just a copy of this.